### DISCUSSION TITLE
Vocs vs Nextra

### DISCUSSION THREAD

**Discussion by hariria** [wevm/vocs#74] [General]
Hi all, just wanted to start off by saying amazing work on Vocs, it looks super clean. I was curious though why sites like viem moved away from Nextra and adopted Vocs instead. Would love to hear your insights!

**Comment by tmm** [comment]
Thanks! For context, Viem never used Nextra (was always VitePress). [Wagmi](https://github.com/wevm/wagmi) used Nextra until v2 shipped recently.

All docs frameworks more or less converge to the same features eventually. With that in mind, we created Vocs because we like VitePress, like React, and have opinions about doc design after maintaining a couple popular OSS projects (Viem + Wagmi). It seemed like a React-based Vite docs framework should exist! We also use Vocs every day so care about keeping things maintained (low issue and open PRs counts just like our other OSS projects) as well as minimal with only important core features.

**Reply by tmm** [reply]
@jxom can chime in with any additional thoughts he might have :)
